test/test_complate.js

```diff
@@ -7,17 +7,32 @@ suite("complate extension");
 
 test("rendering", async () => {
 	let bundle = new Bundle(fixturePath());
-	let html = await bundle.renderString(`
+	let jsx = `
 import Article from "./article.jsx";
 
-<>
+<div>
```

Comment:
huh, why? I'm guessing this is a debugging artifact?

Reply:
Fragment on top-level leads to an array of ASTs. In order to cope with this `src/complate/index.js` (in this project) needs to check for the return value (possible, but not sure if desired).